src/main/java/gyro/google/compute/SecurityPolicyRule.java

```diff
@@ -0,0 +1,189 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.Set;
+
+import com.google.api.services.compute.Compute;
+import com.google.api.services.compute.model.Operation;
+import gyro.core.GyroUI;
+import gyro.core.resource.Resource;
+import gyro.core.resource.Updatable;
+import gyro.core.scope.State;
+import gyro.core.validation.Required;
+import gyro.core.validation.ValidStrings;
+import gyro.google.Copyable;
+
+public class SecurityPolicyRule extends ComputeResource
+    implements Copyable<com.google.api.services.compute.model.SecurityPolicyRule> {
+
+    private String description;
+    private Integer priority;
+    private String action;
+    private Boolean preview;
+    private SecurityPolicyRuleMatcher match;
+
+    /**
+     * The description of the security policy rule.
+     */
+    @Updatable
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    /**
+     * The priority of the security policy rule.
+     */
+    @Required
+    public Integer getPriority() {
+        return priority;
+    }
+
+    public void setPriority(Integer priority) {
+        this.priority = priority;
+    }
+
+    /**
+     * The action to take for this rule. (Required)
+     */
+    @Updatable
+    @Required
+    @ValidStrings({ "allow", "deny(403)", "deny(404)", "deny(502)" })
+    public String getAction() {
+        return action;
+    }
+
+    public void setAction(String action) {
+        this.action = action;
+    }
+
+    /**
+     * The preview flag indicates that this rule is not enforced.
+     */
+    @Updatable
+    public Boolean getPreview() {
+        return preview;
+    }
+
+    public void setPreview(Boolean preview) {
+        this.preview = preview;
+    }
+
+    /**
+     * The match for this rule. (Required).
+     */
+    @Updatable
+    @Required
+    public SecurityPolicyRuleMatcher getMatch() {
+        return match;
+    }
+
+    public void setMatch(SecurityPolicyRuleMatcher match) {
+        this.match = match;
+    }
+
+    @Override
+    public String primaryKey() {
+        return "" + getPriority();
+    }
+
+    com.google.api.services.compute.model.SecurityPolicyRule toSecurityPolicyRule() {
+        com.google.api.services.compute.model.SecurityPolicyRule policyRule = new com.google.api.services.compute.model.SecurityPolicyRule();
+        policyRule.setAction(getAction());
+        policyRule.setDescription(getDescription());
+        policyRule.setPriority(getPriority());
+        policyRule.setPreview(getPreview());
+        policyRule.setMatch(match.toSecurityPolicyRuleMatcher());
```

Comment:
use the getter than directly using the variable
```suggestion
        policyRule.setMatch(getMatch.toSecurityPolicyRuleMatcher());
```